run.go

```diff
@@ -92,6 +92,12 @@ func run(context *cli.Context) error {
 		if err != nil {
 			return fmt.Errorf("Process state %s: %s", ps.String(), err)
 		}
+
+		// delete container's resources
+		err = delete(pod.GetAllContainers()[0].ID(), true)
```

Comment:
that's not possible, if the container is not created ```create``` will fail returning an error